tests/test_epics_motor.py

```diff
@@ -103,3 +104,25 @@ def plan():
         yield from close_run()
 
     RE(plan())
+
+@using_fake_epics_pv
+def test_resume_pause_stop():
+    m = fake_motor()
+    m.stop()
+    assert m.motor_spg.get(as_string = True) == 'Stop'
```

Comment:
I feel like the most pedantic person right now, but the convention is to not put the spaces around the `=` sign when they are function arguments, e.g.
`m.motor_spg.get(as_string=True)`